### PR TITLE
Add "Magnify Image" section in "Microsoft Edge Privacy Whitepaper"

### PR DESCRIPTION
**Pull request by mikehoffms**
PR is **on-hold** & Draft until feature ships in public Edge Stable, on by default.

Rendered article section for review: 
* **Microsoft Edge Privacy Whitepaper** > **Magnify Image**
   * https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image
   * [Planned live](https://learn.microsoft.com/microsoft-edge/privacy-whitepaper/#magnify-image)
* **Develop for the web with Microsoft Edge** > **Zoom an image in an article**
   * https://review.learn.microsoft.com/microsoft-edge/develop-web-microsoft-edge?branch=pr-en-us-3234#zoom-an-image-in-an-article
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/develop-web-microsoft-edge#zoom-an-image-in-an-article)

AB#52714034

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:1]: @sjdallst2 due to github ui, can't directly add you to Reviewers list, but when you post any comment in this PR, that adds you to Reviewers list in main page of PR.

**Comment by sjdallst2** [comment]
ack

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:1]: It appears that you posting a comment on main page of PR doesn't add you to Reviewers list, but I am now seeing you here in the "at" list.  I believe now that if you post a comment here in the Files Changed page of the PR, that adds you to the Reviewers list.  fyi / to figure out PR UI details.  @sjdallst2

Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:1]: i was now able to add @sjdallst2 to the Reviewers list on main page of PR, probably b/c Sam is now member of MicrosoftDocs org, or maybe b/c Sam posted a comment on main page of PR.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:548]: keep or delete screenshot?
@apryzgoda @sjdallst2 
rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:540]: is the menuitem "Magnify image**, or **Magnify Image**? 
trend is toward Sentence case, not Title Case
@sjdallst2 

rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:546]: what's the button label for AI-based image enhance?  
is it settled enough? i think even if in flux, spec'ing a label here would be close enough, can update later if changes
@sjdallst2

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:550]: what's the button label for AI-based image enhance?  
is it settled enough? i think even if in flux, spec'ing a label here would be close enough, can update later if changes
@sjdallst2 

rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:557]: is there a gui nav path instead of having to use command-line flag?  this article is for end users, not devs.
@sjdallst2 

rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:561]: where in edge Settings can you turn off Ctrl-Ctrl shortcut?
@sjdallst2 

rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

File: microsoft-edge/privacy-whitepaper/index.md
Comment (sjdallst2) [reply microsoft-edge/privacy-whitepaper/index.md:540]: The menu item is Magnify image, so this should be fixed.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (sjdallst2) [reply microsoft-edge/privacy-whitepaper/index.md:546]: WebNN can be enabled by going to edge://flags#web-machine-learning-neural-network

File: microsoft-edge/privacy-whitepaper/index.md
Comment (sjdallst2) [reply microsoft-edge/privacy-whitepaper/index.md:561]: edge://settings/content/MagnifyImage will take you directly to the setting.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (sjdallst2) [reply microsoft-edge/privacy-whitepaper/index.md:550]: The label is curently "AI enhance". As far as I know, there are no plans to change this label.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:540]: i changed 1 instance:
right-click an image and then select **Magnify image**

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:546]:
```
To change the WebNN setting, go to `edge://flags`, 
search for **Enables WebNN API**, and then select 
**Default**, **Enabled**, or **Disabled**.  This setting 
enables the Web Machine Learning Neural Network (WebNN) 
API.
```

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:548]: i find the screenshot helpful & interesting, keeping.  if there were too many, could remove it in that case.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:557]: .md file doesn't contain "command line" or "command-line".  

this article (for users) is becoming more separate - elsewhere in the Edge Dev docset (for devs) -- mentions Edge command-line options.
command line: 24 hits in 18 files
command-line: 43 hits in 20 files
eg the wording in: 
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/09/devtools-117#new-experiment-explain-console-errorswarnings-with-copilot-in-sidebar

That example shows two ways: edge://flags flag, & start Edge w/ command-line option.  That example has clear specific wording to possibly use here.

Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:557]: result:
```
To turn off the Magnify Image feature, use 
the command-line flag `msEdgeImageMagnifyUI`.  
For example, run Microsoft Edge from the 
command line using the `enable-features` 
command-line option, as follows:

msedge.exe --enable-features=msEdgeImageMagnifyUI
```

Comment (sjdallst2) [reply microsoft-edge/privacy-whitepaper/index.md:557]: Thanks for finding out how to write this out. The only issue is that to turn off the feature you would use --disable-features.

To turn off the Magnify Image feature, use 
the command-line flag `msEdgeImageMagnifyUI`.  
For example, run Microsoft Edge from the 
command line using the `disable-features` 
command-line option, as follows:

msedge.exe --disable-features=msEdgeImageMagnifyUI

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:561]: result:
```
To turn off the **Ctrl-Ctrl** keyboard 
shortcut, go to `edge://settings/content/MagnifyImage`.  
Or, select **Settings and more** (![Settings and more 
icon](./index-images/settings-and-more-icon.png)) 
> **Cookies and site permissions** > **Magnify image**, 
and then use the **Enable Magnify keyboard shortcut** 
toggle.
```

Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:561]: it seems practical to give both the shortcut url and the gui nav path - seems robust, lots of clues for future maintainers & users, painting an adequate picture, w/o having to show screenshot
image
I'd want a screenshot before that anyway, if we do screenshots of Settings for nav:
image

yet another approach is to use Search box in upper left - that approach is probably not used (described) in this Privacy article, but I'll consider, in a later global update.

**Review by sjdallst2 — APPROVED** [review]

**Review by mikehoffms — COMMENTED** [review]
The PR LGTM, per Writer/Editor review, after pushing commits.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (apryzgoda) [reply microsoft-edge/privacy-whitepaper/index.md:624]: We don't usually put features in the whitepaper until they're released to stable and enabled by default. I think that the screenshot should be removed, and we should wait to post this until it's enabled by default.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:638]: This should be removed. We don't usually tell people how to disable flags. They should be turning the feature on and off via settings. Also, this should be irrelevant once the feature is enabled by default.

```suggestion

```

File: microsoft-edge/privacy-whitepaper/index.md
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:632]: This section should be moved above the AI section for clarity.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:627]: Remove because this isn't needed when the feature is enabled by default.

```suggestion
```

**Review by apryzgoda — CHANGES_REQUESTED** [review]

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:627]: removed png & lines.  @sjdallst2 - if i should restore anything, let me know

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:624]: png removed per other comment.

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:632]: the revised outline at the moment:
```
## Magnify Image
#### Turning off the feature
#### AI-based image enhancement
```

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:638]: removed lines. @sjdallst2 - if i should restore anything, let me know

**Review by sjdallst2 — APPROVED** [review]

File: microsoft-edge/privacy-whitepaper/index.md
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:616]: Changed for consistency

```suggestion
To use the Magnify Image feature, right-click an image and then select **Magnify image**, or press **Ctrl-Ctrl** while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
```

File: microsoft-edge/privacy-whitepaper/index.md
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:628]: Where is the AI enhance button? How do users access it?

**Review by apryzgoda — CHANGES_REQUESTED** [review]

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:628]: image
```
When you click the **AI enhance** button at the bottom of the popup window, AI-based image enhancement is done on your device by using a model that's downloaded from Microsoft servers.
```
@apryzgoda

**Review by apryzgoda — APPROVED** [review]

**Comment by mikehoffms** [comment]
Closing Without Merging.  Will create equivalent PR in Legal repo.  See top of related PR for links:
https://github.com/MicrosoftDocs/edge-developer/pull/3177 (Remove Privacy Whitepaper from edge-developer repo)

**Pull request closed without merging**
